tests/testthat/test-redcap-event-read.R

```diff
@@ -0,0 +1,154 @@
+library(testthat)
+
+read_events <- function(path) {
+  full_path   <- system.file(path, package = "REDCapR")
+  if (!file.exists(full_path))
+    stop("The expected file `", full_path, "` was not found.")  # nocov
+
+  col_types <- readr::cols(
+    event_name         = readr::col_character(),
+    arm_num            = readr::col_integer(),
+    unique_event_name  = readr::col_character(),
+    custom_event_label = readr::col_character(),
+    event_id           = readr::col_double()
+  )
+
+  full_path |>
+    readr::read_csv(col_types = col_types)
+}
+
+empty_data_frame <-
+  tibble::tibble(
+    event_name         = character(0),
+    arm_num            = integer(0),
+    unique_event_name  = character(0),
+    custom_event_label = character(0),
+    event_id           = double(0)
+  )
```

Comment:
I love how you return an empty dataset that has identical columns as a populated dataset.